src/Uno.UI.Tests/Windows_UI_XAML_Controls/Frame/Given_Frame.cs

```diff
@@ -46,9 +47,92 @@ public void When_RemovedPage()
 			Assert.AreEqual(myPage3, SUT.Content);
 			Assert.IsNull(myPage2.Frame);
 		}
+
+		internal static NavigateOrderTracker _navigateOrderTracker = null;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Change the visibility of '_navigateOrderTracker' or make it 'const' or 'readonly'.](https://app.codacy.com/gh/unoplatform/uno/pullRequest?prid=4826531)